facebook/ThreatExchange
# [OpenSource] Create dummy package for all-in-one py code

This PR creates an empty package that will hold the all-in-one scripts.

I went back and forth on various names. I think the best thing to do is to use the existing naming convention from the pytx package in this same repo. 

I copied a lot of the conventions from pytx, and hopefully got all the references.

Test Plan:
1. Set up a python virtual environment
```
$ source ~/.venv/threatexchange_install/bin/activate
```
2. Install from testpypi
```
$ python3 -m pip install --index-url https://test.pypi.org/simple/ pytx3
```
3. Verify it's there
```
$ python3
>>> import pytx3
```
